src/assets/css/librarybrowser.css

```diff
@@ -439,10 +439,43 @@
     background-size: cover;
     background-repeat: no-repeat;
     background-position: center;
+    background-attachment: fixed;
     height: 50vh;
     position: relative;
 }
 
+.layout-desktop .itemBackdrop::after,
+.layout-tv .itemBackdrop::after {
+    content: "";
+    width: 100%;
+    height: 100%;
+    background: #000000ad;
+    background: rgba(0, 0, 0, 0.65);
```

Comment:
uh?

Reply:
It's from aivit's code for this.

Not really needed nowadays to have the hex fallback, but it doesn't hurt (I can remove it tonight if needed).